iaso/tests/api/test_orgunits.py

```diff
@@ -1196,7 +1196,7 @@ def test_org_unit_search_only_direct_children_true(self):
 
         jedi_squad_endor_2_children.save()
 
-        with self.assertNumQueries(6):
+        with self.assertNumQueries(8):
```

Comment:
You have a N+1 here.

I suggest this to fix it https://github.com/BLSQ/iaso/pull/1547